doc/glossary.rst

```diff
@@ -16,13 +16,21 @@
 
         Bucket
                 In the context of :term:`RGW`, a bucket is a group of objects.
-                Buckets are grouped into zones, and zones are grouped into
-                zonegroups. In an filesystem-based analogy in which objects are
-                the counterpart of files, buckets are the counterpart of
+                In an filesystem-based analogy in which objects are the
```

Comment:
This seems stilted, but for sure s/an/a/

Reply:
Yep. That's an artifact of a sentence that originally lacked the adjective "filesystem-based", a sentence in which the indefinite article was yoked to its noun.